osbrain/__init__.py

```diff
@@ -1,4 +1,18 @@
 import os
+import sys
+from pathlib import Path
+
+# While Python 3.4 is supported...
+if sys.version_info < (3, 5):
+    def mkdir(directory, exist_ok=True, parents=True, **kwargs):
```

Comment:
It seems that only the `directory` parameter is used in the function. What do the others do?